templates/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-ENV["RACK_ENV"] = "test"
+ENV["RAILS_ENV"] ||= "test"
```

Comment:
If the tests are run in `development` env then this would prevent loading gems in the `test` gemset.